src/controller/python/chip/clusters/command.cpp

```diff
@@ -248,8 +248,16 @@ PyChipError pychip_CommandSender_SendBatchCommands(void * appContext, DeviceProx
     VerifyOrReturnError(device->GetSecureSession().HasValue(), ToPyChipError(CHIP_ERROR_MISSING_SECURE_SESSION));
     auto remoteSessionParameters = device->GetSecureSession().Value()->GetRemoteSessionParameters();
     CommandSender::ConfigParameters config;
+
     config.SetRemoteMaxPathsPerInvoke(remoteSessionParameters.GetMaxPathsPerInvoke());
 
+    // TODO(#30986): Need to create a separate pychip_CommandSender_TestOnlySendBatchCommands so that we perform
+    // operations that is very clear at callsite that violating certain aspects like setting this MaxPathsPerInvoke
+    // to a number other than what is reported by the remote node is allowed. Right now the only user of this
+    // function is cert test script. To implement pychip_CommandSender_TestOnlySendBatchCommands in a clean way
+    // we need to move away from the variadic arguments.
+    config.SetRemoteMaxPathsPerInvoke(std::numeric_limits<uint16_t>::max());
```

Comment:
Does this just override the SetRemoveMaxPathsPerInvoke call from earlier? Does that mean that ALL the calls to SendBatchCommands will use this value? Why not just send that value in as the config parameter?

Reply:
> Does this just override the SetRemoveMaxPathsPerInvoke call from earlier?
Yes it does.

> Does that mean that ALL the calls to SendBatchCommands will use this value?

>Why not just send that value in as the config parameter?

As the comment states I want anything that sets this via a parameter calling `pychip_CommandSender_TestOnlySendBatchCommands` (not the TestOnly in the name). Today only tests call SendBatchCommands, so I am hacking this in for just a couple weeks.

I plan to fix the variadic arguments issue holding me back from easily creating `pychip_CommandSender_TestOnlySendBatchCommands` stuff tomorrow actually so that I can more cleanly do this going forward.

Reply:
Here is a separate PR I have for CFFI stuff, that likely will take some more time before it gets in

https://github.com/project-chip/connectedhomeip/pull/31159

Reply:
That's fine, but why set it to remoteSessionParameters.GetMaxPathsPerInvoke() then immediately set it to max?